hubble-fe/src/stores/utils/index.ts

```diff
@@ -0,0 +1,5 @@
+export function checkIfLocalNetworkOffline(error: any) {
+  if (!error.response) {
+    throw new Error('网络错误，请查看您的本地连接');
```

Comment:
TODO: use i18n instead

Reply:
No i18n support before v1.2.1 here, plan to add it in v1.3.0